tests/integration/update_cluster/compress/in-v1alpha2.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: kops.k8s.io/v1alpha2
```

Comment:
I'd go with something minimal here, to be less affected by various changes in the future.